### ISSUE TITLE
Report isn't generated

### ISSUE THREAD

**Issue by AmandaBSobrinho** [flipkart-incubator/Astra#88]
I installed Astra on Kali Linux 2018.4, and after the scan is finished on CLI (the message "Scan has been completed" shows up), a tab opens on Firefox and I believe it should show the report and the results. However, it shows the message "NoProduct NameurlStatus", and no report. What am I doing wrong? The images with this process follow.

![astra-1](https://user-images.githubusercontent.com/16309667/51495554-eaf04380-1da3-11e9-8f9a-df49f05b598b.png)

![astra-2](https://user-images.githubusercontent.com/16309667/51495556-ee83ca80-1da3-11e9-83b5-a11c7bcb5342.png)

This is the scan.log file:

![astra-3](https://user-images.githubusercontent.com/16309667/51495560-f0e62480-1da3-11e9-8bdc-286efcd24876.png)

**Comment by sagarpo** [comment]
@AmandaBSobrinho Can you try the same from the web interface?

**Comment by AmandaBSobrinho** [comment]
With the web interface, I also have no response after the scan has supposedly started. When I click "Submit" after informing the Product Name, the URL and the Method (the Headers and the Body are left blank), I get a status. 

![captura de tela de 2019-02-05 08-58-33](https://user-images.githubusercontent.com/16309667/52269308-d1fb9c80-2924-11e9-97f3-0590ac32ca99.png)

After clicking "OK", nothing happens. There is no information about the scan, if it has started, ended or anything.

![captura de tela de 2019-02-05 09-06-10](https://user-images.githubusercontent.com/16309667/52269474-52220200-2925-11e9-92c6-440df2aa0095.png)

Clicking on "Reports" gives me the same answer I got before with the CLI.

![captura de tela de 2019-02-05 09-04-43](https://user-images.githubusercontent.com/16309667/52269406-1850fb80-2925-11e9-8ce2-007134e5bd12.png)

Am I using the Web Interface incorrectly?

**Comment by sagarpo** [comment]
Are you using Docker version? Also, can you check if your mongo server is running properly ?

**Comment by AmandaBSobrinho** [comment]
No, I'm not using docker version, I installed it with pip (sudo pip install -r requirements.txt), because I wasn't able to make it work with Docker. I first run nohup mongod &, and then netstat -ntlvp, which gives me the answer:

![captura de tela de 2019-02-05 10-53-54](https://user-images.githubusercontent.com/16309667/52274616-5efa2200-2934-11e9-8738-6001c96be230.png)

Then, I run the command to start Astra (python api.py, inside the API folder).

**Comment by AmandaBSobrinho** [comment]
Oh, and this error is showing up on terminal after I start the scan on the web interface. Maybe it can help.

![captura de tela de 2019-02-05 11-18-27](https://user-images.githubusercontent.com/16309667/52275839-cbc2eb80-2937-11e9-87d6-2fa76689f124.png)

**Comment by sagarpo** [comment]
@AmandaBSobrinho Can you share your flask,python and mongo version?

**Comment by AmandaBSobrinho** [comment]
Sure! Here it is:

Flask version: 0.10.1
Python version: 2.7.15+
Mongo version: v3.4.18

**Comment by sagarpo** [comment]
@AmandaBSobrinho This might be a problem with flask version. Can you update the flask version and try?

**Comment by lucky73** [comment]
I think your mongo is not running. Please check the status for the same. you can see before and after mongod server start and stop.
![4](https://user-images.githubusercontent.com/29023706/53864714-19894d00-3fe5-11e9-9ef3-2981d07699cf.PNG)
![1](https://user-images.githubusercontent.com/29023706/53864716-19894d00-3fe5-11e9-97b2-4363833545ea.PNG)
![2](https://user-images.githubusercontent.com/29023706/53864717-19894d00-3fe5-11e9-94a7-ebd0527fe43d.PNG)
![3](https://user-images.githubusercontent.com/29023706/53864719-1a21e380-3fe5-11e9-8815-271e97d04f10.PNG)

**Comment by AmandaBSobrinho** [comment]
Hi! So, I tried starting MongoDB with "service mongdb start", but it's still not working. In the image below, we can see that MongoDB has been started and is running on port 27017.

![captura de tela de 2019-03-06 16-56-23](https://user-images.githubusercontent.com/16309667/53909660-fb493e80-4030-11e9-8b97-d808f06bb8f9.png)

Then, I run "python api.py" to start the Web Interface, and I click on "Submit" to start the scan.

![captura de tela de 2019-03-06 16-59-35](https://user-images.githubusercontent.com/16309667/53909878-62ff8980-4031-11e9-90ed-155828d45cb2.png)

Again, nothing shows up and when I click on "Reports", I still can't see anything.

![captura de tela de 2019-03-06 17-00-06](https://user-images.githubusercontent.com/16309667/53909919-77dc1d00-4031-11e9-8c67-dc0246c61601.png)

The terminal where I ran "python api.py" shows the same error I mentioned before:

![captura de tela de 2019-03-06 17-01-48](https://user-images.githubusercontent.com/16309667/53909996-9b9f6300-4031-11e9-8fc5-cab01030d16f.png)
